donation_mass/donation.py

```diff
@@ -81,15 +81,18 @@ def validate(self):
         return res
 
     @api.one
```

Comment:
Just because the original done2cancel() method in the donation module is @api.one (ok, it's not a good reason) and to avoid a loop on the donations.

This method doesn't return any special result, so it's not a problem is the result is encapsulated in a list, but I know that it's sometimes a pb (for example when the method returns an action).